Formula/php-spx.rb

```diff
@@ -0,0 +1,19 @@
+require_relative "../lib/php_pecl_formula"
+
+class PhpSpx < PhpPeclFormula
+  extension_dsl "SPX - A simple profiler for PHP"
+
+  homepage "https://github.com/NoiseByNorthwest/php-spx"
+  url "https://github.com/NoiseByNorthwest/php-spx/archive/refs/tags/v0.4.20.tar.gz"
+  sha256 "8de7f8e6137667dbe7e92ba552ccb3b3b3745664efb9af88ece8bda0f58fc94f"
+  license "GPL-3.0"
```

Comment:
Same as above. Use the correct license identifier.

```suggestion
  license "GPL-3.0-or-later"
```